python-sdk/src/astro/sql/operators/base_decorator.py

```diff
@@ -27,7 +27,8 @@ def __init__(
         handler: Function | None = None,
         database: str | None = None,
         schema: str | None = None,
-        response_limit: int = 0,
+        response_limit: int = -1,
```

Comment:
@tatiana Probably too late for the question, but what's the difference between response_limit and response_size?

Reply:
Also, shouldn't we assign default values from RAW_SQL_MAX_RESPONSE_SIZE?

Reply:
@utkarsharma2:
* `response_limit`: raises an exception if the response limit is passed, used by the `get_value_list` dynamic task template. It was introduced by #673, to avoid passing the limit of allowed parallel tasks of Airflow itself. It is a hard limit which is not user-defined. If there are more values than this limit, the DTT will fail.
* `response_size`: "cuts" or "trims" the number of items in the response of the `run_raw_sql` decorator. This was done as a follow up from @kaxil's comment https://github.com/astronomer/astro-sdk/commit/5b93df870dac5787dc1ec4ff11f238b9c360ebb0#r83305025. This PR avoids introducing a breaking change, so we log a warning if the user uses the default backend and does not set this size. If the user sets this size, we cut the response without raising an exception.

Since we do not want to introduce a breaking change, I couldn't just reuse `response_limit`